README.md

```diff
@@ -43,13 +43,21 @@ const wallet = new Wallet({adapter: localforage});
 
 In your file, where you want to execute it :
 
-```
-const { Wallet, EVENTS } = require('@dashevo/wallet-lib');
+```js
+const { Wallet, EVENTS } = require('./src');
```

Comment:
Did you do that by mistake?

Reply:
It's done so they just have to copy and paste, so no, this is really how we want it (as this will be required outside of the wallet-lib folder.